### PR TITLE
🚸remove thumbnail from file-upload section and show bitstream format …

### LINKED ISSUE #2105: UX: Thumbnail is never available during the submission
**Describe the bug**
The upload step in the submission includes a thumbnail but this can never be present at this stage.

**To Reproduce**
1. start a new submission and upload a file
2. the upload step will show a "No thumbnail available" placeholder

**Expected behavior**
It would be better to avoid to include the thumbnail placeholder at all. The extra space could be used to provide more information about the file that are currently hidden in the edit screen (access conditions and extra metadata) or details about technical metadata (format and checksum)

### PR DESCRIPTION
**Pull request by hugo-escire**
Hi @tdonohue , I'm @jtimal  partner. We have been working on this activity and now we will send you the pull request.
## References
Fixes #2105

## Description
Now the thumbnail is hidde and show more information about the bistream.

image

List of changes in this PR:
* Thumbnail was deleted from file-upload section .
* In file-upload section is now showing the bitstream format and checksum.

## Instructions for Reviewers
* Start a new submission
* Upload a file

## Checklist
- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [x] My PR passes [ESLint](https://eslint.org/) validation using yarn lint
- [x] My PR doesn't introduce circular dependencies (verified via yarn check-circ-deps)
- [x] My PR includes [TypeDoc](https://typedoc.org/) comments for all new (or modified) public methods and classes. It also includes TypeDoc for large or complex private methods.
- [x] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [x] If my PR includes new libraries/dependencies (in package.json), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
 - [x] If my PR includes new features or configurations, I've provided basic technical documentation in the PR itself.
 - [x] If my PR fixes an issue ticket, I've [linked them together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

### CODE REVIEW

**Comment by alanorth** [comment]
Tested on DSpace 7.6, works as expected:

<p align="center">
  
</p>

The space is now used much more efficiently.

File: src/app/core/submission/models/workspaceitem-section-upload-file.model.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { SubmissionUploadFileAccessConditionObject } from './submission-upload-file-access-condition.model';
-import { WorkspaceitemSectionFormObject } from './workspaceitem-section-form.model';
+import {SubmissionUploadFileAccessConditionObject} from './submission-upload-file-access-condition.model';
+import {WorkspaceitemSectionFormObject} from './workspaceitem-section-form.model';
```
Comment (alanorth) [inline src/app/core/submission/models/workspaceitem-section-upload-file.model.ts:2]: Unnecessary changes to whitespace in imports here.

File: src/app/submission/sections/upload/file/view/section-upload-file-view.component.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-import { WorkspaceitemSectionUploadFileObject } from '../../../../../core/submission/models/workspaceitem-section-upload-file.model';
-import { isNotEmpty } from '../../../../../shared/empty.util';
-import { Metadata } from '../../../../../core/shared/metadata.utils';
-import { MetadataMap, MetadataValue } from '../../../../../core/shared/metadata.models';
+import {
+  WorkspaceitemSectionUploadFileObject
+} from '../../../../../core/submission/models/workspaceitem-section-upload-file.model';
+import {isNotEmpty} from '../../../../../shared/empty.util';
+import {Metadata} from '../../../../../core/shared/metadata.utils';
+import {MetadataMap, MetadataValue} from '../../../../../core/shared/metadata.models';
```
Comment (alanorth) [inline src/app/submission/sections/upload/file/view/section-upload-file-view.component.ts:9]: Unnecessary changes to whitespace in imports here.

**Comment by hugo-escire** [comment]
Hi @alanorth, I revert the format changes. 
Greetings.

**Review by tdonohue — APPROVED** [review]
👍 Thanks @hugo-escire !  This looks good to me and works.  That said, I pushed up a very minor fix to a code comment (a copy & paste error).  Beyond that, it looks good.
